source/packages/modulo_core/include/modulo_core/Communication/EncodedState.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include <std_msgs/msg/byte_multi_array.hpp>
+
+namespace modulo::core {
+	using EncodedState = std_msgs::msg::ByteMultiArray;
```

Comment:
I prefer a `typedef` to a `using` declaration, although I think it's the same in the end 

```suggestion
	typedef std_msgs::msg::ByteMultiArray EncodedState;
```